sdcm/utils/benchmarks.py

```diff
@@ -12,13 +12,55 @@
 # Copyright (c) 2021 ScyllaDB
 import logging
 import re
+from dataclasses import dataclass, field
+from pathlib import Path
 
 from sdcm.remote import RemoteCmdRunnerBase, shell_script_cmd
+from sdcm.utils.common import ParallelObject
 from sdcm.utils.metaclasses import Singleton
 
 LOGGER = logging.getLogger(__name__)
 
 
+@dataclass
+class ScyllaNodeBenchmarkSysbenchResult:
+    # pylint:disable=too-many-instance-attributes
+    cmd_output: str = field(repr=False)
+    events_per_second: float = field(init=False)
+    latency_min: float = field(init=False)
+    latency_max: float = field(init=False)
+    latency_avg: float = field(init=False)
+    latency_95: float = field(init=False)
```

Comment:
I guess better to add 'p' symbol as it is done in lots of other places:
```
95th percentile -> p95
99th percentile -> p99
```
As a result:
```
latency_p95: float = field(init=False)
```

Reply:
Agreed, better to keep the "p". Fixed.